charms/katib-ui/requirements.in

```diff
@@ -1,8 +1,6 @@
-ops
-oci-image
-serialized-data-interface
-lightkube
+charmed-kubeflow-chisme
 jinja2
```

Comment:
`jinja2` is fetched by chisme, so not needed here.

Without chisme we would actually have to explicitly specify it because lightkube needs it (but doesn't come with it), but that's a different story.